### PR TITLE
Aircash

### PR DESCRIPTION
**Pull request by habib7234**
PERFECT project ❤️ ❤️
Aircoin AirCash airdrop

❤️❤️❤️❤️❤️❤️❤️❤️

### CODE REVIEW

**Comment by prachodayat13** [comment]
Nice project

**Comment by Johnade80** [comment]
Breath cool air
Breath oxygen

**Comment by Kelvinyoungluv** [comment]
Air

**Comment by mbun07** [comment]
Aircash airdrop, 
This big project and real

**Review by Champ11jun1980 — COMMENTED** [review]
Very nice project

**Comment by Champ11jun1980** [comment]
Very nice project

**Comment by Onyxchuzzy** [comment]
This new project is well detailed and is serious. I just hope that whales do not crash it for us. Air, the oxygen crypto will breathe

**Comment by JalesRamsett** [comment]
It's a great project, a really awesome big airdrop and good luck

Pada tanggal Sab, 7 Agt 2021 04.47, Kelvinyoungluv ***@***.***>
menulis:

**Comment by T1TAN2** [comment]
Its a great project

**Comment by habbyr** [comment]
i cant life without airr

**Comment by Namanjain1116** [comment]
Awesome name air

**Comment by fauzizz** [comment]
I'm so sure this project will Gona blow up

**Comment by Alisaki1361** [comment]
Im sure too

**Comment by raohussain** [comment]
good project

**Comment by kevin56778** [comment]
Easy

**Review by kevin56778 — APPROVED** [review]

File: Aircash [hunk 1]
```diff
@@ -0,0 +1 @@
+Aircoin - AirCash airdrop
```
Comment (artiomleo) [inline Aircash:1]:
```suggestion
Aircoin - AirCash Airdrop
```

**Review by artiomleo — CHANGES_REQUESTED** [review]

**Comment by Cryptossaddict** [comment]
This is just a really wonderful crypto project ! Worth great attention and investment !

**Comment by Deborahmagaji20** [comment]
I love Air. I breathe Air

File: Aircash (hunk 1 above)
Comment (TonyEdwards) [reply Aircash:1]: Thanks

Comment (skrivcoin) [inline Aircash:1]: Good!

**Comment by JD536767** [comment]
Unsubscribe

**Pull request closed without merging**